src/react-apps/applications/ux-editor/src/actions/formDesignerActions/formDesignerActionDispatcher.ts

```diff
@@ -46,6 +47,14 @@ export interface IFormDesignerActionDispatchers
   addActiveFormContainerRejected: (
     error: Error,
   ) => FormDesignerActions.IAddActiveFormContainerRejected;
+  createRepeatingGroup: (
+    id: string,
+  ) => FormDesignerActions.ICreateRepeatingGroupAction;
+  createRepeatingGroupFulfilled: (
+  ) => FormDesignerActions.ICreateRepeatingGroupFulfilled;
```

Comment:
Ref comment about not extending Action interface. If you're removing the interface, just use this:
`createRepeatingGroupFulfilled: () => Action;`